<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuplicatesDialog</class>
 <widget class="QDialog" name="DuplicatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="actionCombo">
     <item>
      <property name="text">
       <string>Remove old elements on duplicates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remove non object elements on objects</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="strategyLabel">
     <property name="text">
      <string>Strategy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="strategyCombo">
     <item>
      <property name="text">
       <string>Remove from sublayer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remove from layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remove everywhere</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="heursticLabel">
     <property name="text">
      <string>Heurstic</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="heursticCombo">
     <item>
      <property name="text">
       <string>Extract date from KERG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extract date from element</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="typeCombo">
     <item>
      <property name="text">
       <string>Text layers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line layers</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="layerLabel">
     <property name="text">
      <string>Layer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="layerCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sublayerLabel">
     <property name="text">
      <string>Sublayer</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="sublayerCombo"/>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="distanceSpin">
       <property name="specialValueText">
        <string>Match exactly</string>
       </property>
       <property name="prefix">
        <string>Search in </string>
       </property>
       <property name="suffix">
        <string> m radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DuplicatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DuplicatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
